pyproject.toml

```diff
@@ -75,6 +75,7 @@ opentelemetry-instrumentation = { version = "*", optional = true }
 sqlalchemy = { version = "*", extras = ["asyncio"], optional = true }
 opentelemetry-exporter-otlp = { version = ">=1.20.0", optional = true }  # log body serialization is required
 pyyaml = { version = "*", optional = true }
+langchain = { version = "*", optional = true }
```

Comment:
What about `langchain-openai`? it doesn't seem to be installed by default and I got import errors until I added it manually

Reply:
The error 
```
INFO:chatsky.core.script_parsing:Loaded file /home/rami1996/ui/my_project/bot/scripts/build.yaml
INFO:chatsky.messengers.common.interface:Attachments directory for LongpollingInterface messenger interface is None, so will be set to tempdir and attachment data won't be cached locally!
Exception: [ModuleNotFoundError("No module named 'langchain_openai'")]

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rami1996/ui/my_project/./app.py", line 19, in <module>
    main()
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/click/core.py", line 1161, in __call__
    return self.main(*args, **kwargs)
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/click/core.py", line 1082, in main
    rv = self.invoke(ctx)
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/click/core.py", line 1443, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/click/core.py", line 788, in invoke
    return __callback(*args, **kwargs)
  File "/home/rami1996/ui/my_project/./app.py", line 14, in main
    pipeline = Pipeline.from_file(file = script_path, custom_dir = Path(script_path).parent.parent / "custom")
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/chatsky/core/pipeline.py", line 182, in from_file
    pipeline = JSONImporter(custom_dir=custom_dir).import_pipeline_file(file)
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/chatsky/core/script_parsing.py", line 263, in import_pipeline_file
    return self.replace_resolvable_objects(pipeline)
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/chatsky/core/script_parsing.py", line 232, in replace_resolvable_objects
    return {k: (self.replace_resolvable_objects(v)) for k, v in obj.items()}
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/chatsky/core/script_parsing.py", line 232, in <dictcomp>
    return {k: (self.replace_resolvable_objects(v)) for k, v in obj.items()}
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/chatsky/core/script_parsing.py", line 232, in replace_resolvable_objects
    return {k: (self.replace_resolvable_objects(v)) for k, v in obj.items()}
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/chatsky/core/script_parsing.py", line 232, in <dictcomp>
    return {k: (self.replace_resolvable_objects(v)) for k, v in obj.items()}
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/chatsky/core/script_parsing.py", line 229, in replace_resolvable_objects
    args, kwargs = self.parse_args(obj[key])
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/chatsky/core/script_parsing.py", line 196, in parse_args
    value = self.replace_resolvable_objects(value)
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/chatsky/core/script_parsing.py", line 232, in replace_resolvable_objects
    return {k: (self.replace_resolvable_objects(v)) for k, v in obj.items()}
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/chatsky/core/script_parsing.py", line 232, in <dictcomp>
    return {k: (self.replace_resolvable_objects(v)) for k, v in obj.items()}
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/chatsky/core/script_parsing.py", line 230, in replace_resolvable_objects
    return self.resolve_string_reference(key)(*args, **kwargs)
  File "/home/rami1996/.cache/pypoetry/virtualenvs/chatsky-ui-Yhfmz2Co-py3.10/lib/python3.10/site-packages/chatsky/core/script_parsing.py", line 181, in resolve_string_reference
    raise JSONImportError(f"Could not import {obj}") from Exception(exceptions)
chatsky.core.script_parsing.JSONImportError: Could not import langchain_openai.ChatOpenAI
Read the guide on Pipeline import from file: https://deeppavlov.github.io/chatsky/user_guides/pipeline_import.html
```

Reply:
We do not include langchain-<provider> packages by default to prevent dependencies versions mismatch. As it is shown in tutorials you need to install them by yourself like so `pip install chatsky[llm] langchain-openai`